src/features/room/create/ui/EditRoomFormModal.module.css

```
@@ -0,0 +1,365 @@
+}
+
+.closeButton:focus-visible {
+  outline: 2px solid #5b8cff;
+  outline-offset: 2px;
+}
+
+.closeIcon {
+  display: block;
+  width: 20px;
+  height: 17px;
+  background-color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)